k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/prometheus-eks-windows-exporter.yaml

```diff
@@ -0,0 +1,177 @@
+                  ]
+                },
+                {
+                  "source_labels": ["job"],
+                  "label_matcher": "^pod_exporter$",
+                  "dimensions": [["instance"]],
+                  "metric_selectors": [
+                    "^windows_netframework_clrmemory_allocated_bytes_total$"
+                  ]
+                },
+                {
+                  "source_labels": ["job"],
+                  "label_matcher": "^pod_exporter$",
+                  "dimensions": [["instance"]],
+                  "metric_selectors": [
+                    "^windows_netframework_clrmemory_finalization_survivors$"
+                  ]
+                },
+                {
+                  "source_labels": ["job"],
+                  "label_matcher": "^pod_exporter$",
+                  "dimensions": [["instance"]],
+                  "metric_selectors": [
+                    "^windows_netframework_clrmemory_collections_total$"
+                  ]
+                }
+              ]
+            }
+          }
+        },
+        "force_flush_interval": 5
+      }
+    }
+kind: ConfigMap
+metadata:
+  name: prometheus-cwagentconfig
+  namespace: amazon-cloudwatch
+
+---
+# create configmap for prometheus scrape config
+apiVersion: v1
+data:
+  # prometheus config
+  prometheus.yaml: |
+    global:
+      scrape_interval: 1m
+      scrape_timeout: 10s
+    scrape_configs:
+      - job_name: pod_exporter
+        kubernetes_sd_configs:
+        - role: pod
+
+kind: ConfigMap
+metadata:
+  name: prometheus-config
+  namespace: amazon-cloudwatch
+
+---
+# create cwagent service account and role binding
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cwagent-prometheus
+  namespace: amazon-cloudwatch
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cwagent-prometheus-role
+rules:
+  - apiGroups: [""]
+    resources:
+    - nodes
+    - nodes/proxy
+    - services
+    - endpoints
+    - pods
+    verbs: ["get", "list", "watch"]
+  - apiGroups:
+    - extensions
+    resources:
+    - ingresses
+    verbs: ["get", "list", "watch"]
+  - nonResourceURLs: ["/metrics"]
+    verbs: ["get"]
+
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cwagent-prometheus-role-binding
+subjects:
+  - kind: ServiceAccount
+    name: cwagent-prometheus
+    namespace: amazon-cloudwatch
+roleRef:
+  kind: ClusterRole
+  name: cwagent-prometheus-role
+  apiGroup: rbac.authorization.k8s.io
+
+---
+# Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cwagent-prometheus
+  namespace: amazon-cloudwatch
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cwagent-prometheus
+  template:
+    metadata:
+      labels:
+        app: cwagent-prometheus
+    spec:
+      containers:
+        - name: cloudwatch-agent
+          image: public.ecr.aws/cloudwatch-agent/cloudwatch-agent:1.300042.1b746
```

Comment:
Could you update this script https://github.com/aws-samples/amazon-cloudwatch-container-insights/blob/main/container-insights-manifest-update.sh so that this YAML gets updated with the latest version of the agent as we release it